### PR TITLE
Fix coverage generation

### PR DESCRIPTION
**Pull request by antipeon**

### CODE REVIEW

**Comment by antipeon** [comment]
in `bin` target was .o file: Screenshot (70)
now test executable: Screenshot (68)
and coverage is shown: Screenshot (69)

**Comment by antipeon** [comment]
for .so files remains unchanged: 
Screenshot (71)
coverage was already shown:
Screenshot (72)

File: server/src/printers/NativeMakefilePrinter.cpp [hunk 1]
```diff
@@ -354,7 +354,14 @@ namespace printer {
         fs::path testExecutablePath = getTestExecutablePath(sourcePath);
 
         auto rootLinkUnitInfo = buildDatabase->getClientLinkUnitInfo(rootPath);
-        declareTarget("bin", { FORCE }, { stringFormat("echo %s", sharedOutput.value()) });
+
+        fs::path coverageInfoBin = sharedOutput.value();
+        if (!Paths::isLibraryFile(coverageInfoBin))
+        {
```
Comment (ladisgin) [inline server/src/printers/NativeMakefilePrinter.cpp:360]: Use same style as other file. Move open brackets on same line as if statement.

**Review by ladisgin — CHANGES_REQUESTED** [review]
squash commits

**Review by ladisgin — APPROVED** [review]

**Pull request merged**